engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -162,7 +162,7 @@ private void processInput(EntityRef entity, CharacterMovementComponent character
         Quat4f viewRotation;
         switch (characterMovementComponent.mode) {
             case WALKING:
-                if (!playerCamera.isTracked()) {
+                if (!config.getRendering().isVrSupport()) {
```

Comment:
Other than this I think we are good to go. Please take care of this so that I can greenlight everything. At this stage I don't think we should wait anymore for further reviews from Josharias, obviously he's very busy right now.